recipes/gflags/all/test_package/CMakeLists.txt

```diff
@@ -1,9 +1,10 @@
 cmake_minimum_required(VERSION 3.1)
 project(test_package)
 
-
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+conan_basic_setup(TARGETS)
+
+find_package(gflags REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
-target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+target_link_libraries(${PROJECT_NAME} gflags::gflags)
```

Comment:
but it's still namespaced here?

Reply:
Yes, I can change to non namespaced one.

Actually official config file provides either `gflags` or `gflags::gflags` imported target.
`gflags` is the default imported target unless if `gflags_USE_TARGET_NAMESPACE` is set to True before find_package.
https://github.com/gflags/gflags/blob/master/cmake/config.cmake.in#L112-L117

So in a transparent integration, you would do:

```cmake
find_package(gflags)
target_link_libraries(mytarget gflags)
```

or

```cmake
set(gflags_USE_TARGET_NAMESPACE ON)
find_package(gflags)
target_link_libraries(mytarget gflags::gflags)
```

Reply:
sounds complicated... why did they do it? never mind, leave it as is :)

Reply:
whatta mess!

Reply:
~How is this related to the namespace feature?~ nvm it's the C++ namespace

Reply:
if it's C++ namespace, then this CMake fix shouldn't be needed?

Reply:
What are you talking about?

Reply:
This recipe has a `namespace` option.... It configures a header 

https://github.com/gflags/gflags/blob/e171aa2d15ed9eb17054558e0b3a6a413bb01067/CMakeLists.txt#L349

The CMake is coded (or lack there of) by upstream and is valid

Excuse my confusion from last night